src/fvm_multicell.hpp

```diff
@@ -233,8 +247,8 @@ class fvm_multicell {
 //////////////////////////////// Implementation ////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////
 
```

Comment:
Yup, let's bake in the default for now and add it back as part of a backend policy class.